src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+temp-message-dir = /var/opt/ctd-omega-services
```

Comment:
There's no such directory for (standard) macos.

Reply:
I have changed this to a relative path for now so it should create the folder where the application is being run. This is not what we want in a production environment but we can come back to that later.

Reply:
Confirmed. Perhaps we could add `temp-messages` to the `.gitignore` file?

Reply:
Done.

Reply:
Confirmed.